akka-persistence/src/main/scala/akka/persistence/Persistence.scala

```diff
@@ -102,6 +102,19 @@ trait PersistenceIdentity {
 }
 //#persistence-identity
 
+trait PersistenceRecovery {
+  //#persistence-recovery
+  /**
+   * Called when the persistent actor is started for the first time.
+   * The returned [[Recover]] object defines how the Actor will recover its persistent state before
+   * handling the first incoming message.
+   *
+   * To skip recovery completely return `NoRecovery()`.
+   */
+  def recovery: Recover = Recover() // TODO I don't like the names here somehow; maybe recovery = Recovery(to...) ?
```

Comment:
TODO I'm not happy with names here. 
Maybe add a trait `Recovery`?
I'm tempted to provide a builder for it as well (for java users), `Recovery.builder().toSequenceNr(...)....`